### ISSUE TITLE
Video starts to play when i am in the subs page

### ISSUE THREAD

**Issue by Joshua64** [raingart/Nova-YouTube-extension#91]
Chrome 113.0.5672.127, windows 10 with latest builds and updates, nova script 0.42.0+9ad24c4

Hello
I saw a video in youtube and when the final of the video i clicked in subscriptions tab but the video started to play again, but in second plan, the subs page was fully loaded but the video was still playing and stopped only when i reloaded the page, i am the guy with the auto pause and stop video preload issue, and i am still having this issue now with this new one.

**Comment by raingart** [comment]
All functions are so unstable. As a developer, I constantly observe how something stops working. And if the error can't be reproduced regularly, I just ignore it.

The problem is that YouTube is updated in parts. And plugins are also updated in parts.
i have 3 ways to update/restart code

1. complete restart of the plugin
2. partial restart
3. event restart

and they all work like crap. There are many more of them.
For example, there is a plugin for displaying the number of videos on a channel. And YouTube updates the channel id without any notice. There is no way to know if the id has been updated or not. Can be updated immediately after the player. Or for example after changing the url. I had to put a stupid stub half a second for it to wait (fingers crossed) for the channel id to update.
And comparing the new id with the old one is also not good. The channel may remain the same. Just a different video. And there are many such examples.
Also the launch of plugins is sequential. That is, some can be in time before loading the player, while others are no longer there.
In general, the fact that nova generally works this miracle.
If the bug is not an eyesore, then it is better to ignore

I'll throw a link to another version of the beta. And let's both hope that it will work.



https://gist.github.com/raingart/eae39323b6751a9ea3b40c87dc8bbef6/raw/5de5cb9ca967703526d43b26fe9aebbd37825379/nnn.user.js
